### PR TITLE
docs: add proguard annotation to quickstart

### PR DESCRIPTION
**Pull request by sdenaci**
Addresses Proguard build difficulty in #17 

How to test:
Follow instructions in quickstart
Build in android release mode, with proguard minifyEnabled on
App should work normally

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -92,6 +92,15 @@ For advanced setup please refer to [cookbook](./cookbook.md)
          }
          ```
 
+      3. If you are using ProGuard:
+         in `proguard-rules.pro` add
+
+         ```java
+         -keepclassmembers class [YOUR_GOOGLE_APPLICATION_NAME].BuildConfig {
+            public static <fields>;
+         }
```
Comment (maxkomarychev) [inline docs/quickstart.md:101]: is this the literal fragment that needs to be copypasted or does it need to list every single field?

Comment (maxkomarychev) [reply docs/quickstart.md:101]: can `https://developer.android.com/studio/build/shrink-code#keep-code` this be used instead?

```
-keep public class MyClass
```

File: docs/quickstart.md [hunk 2]
```diff
@@ -92,6 +92,15 @@ For advanced setup please refer to [cookbook](./cookbook.md)
          }
          ```
 
+      3. If you are using ProGuard:
+         in `proguard-rules.pro` add
+
+         ```java
+         -keepclassmembers class [YOUR_GOOGLE_APPLICATION_NAME].BuildConfig {
```
[the remark this answers is not in the document]
Comment (sdenaci) [reply docs/quickstart.md]: that is much better! thank you for suggestion! committed it :D

**Comment by maxkomarychev** [comment]
thank you!

**Pull request merged**